### PR TITLE
Typo corrections and broken link update

### PR DESCRIPTION
**Pull request by tim-arney**

### CODE REVIEW

**Review by lsf37 — APPROVED** [review]

File: projects/sel4/frequently-asked-questions.md [hunk 1]
```diff
@@ -628,7 +628,7 @@ There are two recommended ways to do this.
 
 For build instructions, and how to get started, see the
 [Getting started](/GettingStarted) page.
-Also, UNSW's [Advanced Operating Systems course](https://cs9242.web.cse.unsw.edu.au/) has an extensive project component that
+Also, UNSW's [Advanced Operating Systems course](http://www.cse.unsw.edu.au/~cs9242) has an extensive project component that
```
Comment (kent-mcleod) [inline projects/sel4/frequently-asked-questions.md:631]: Can this updated URL still use `https`? The new https url resolves for me.

Comment (tim-arney) [reply projects/sel4/frequently-asked-questions.md:631]: HTTPS wasn't resolving when I first tested but seems to be working now, will update and issue another pull request.

Comment (tim-arney) [reply projects/sel4/frequently-asked-questions.md]: Yes, HTTPS wasn't working when I first updated the link, but seems ok now.

**Review by kent-mcleod — APPROVED** [review]
Thanks!

**Pull request merged**